source/Classroom/Track/Collecting_Data/a_guide_for_tracking_data_with_keen_io.md

```diff
@@ -1,49 +1,45 @@
+Create a <a href="https://keen.io/signup">Keen IO account</a> (if you don't already have one)
 
+{% anchor h4 %}
+Step 2: Activate SendGrid's Event Notification App
+{% endanchor h4 %}
 
-**Step 2: Activate SendGrid's Event Notification App**
-
-First click on <a href="http://app.sendgrid.com/settings/mail_settings">Mail Settings</a> within Settings in the left column. Next, locate and activate the Event Notification Setting.
-
-![SendGrid- Mail Settings]({{root_url}}/images/keen/sendgrid_mailsettings_eventnotification.png)
-
-Enter the Keen HTTP POST URL, exampled below, as the endpoint where SendGrid will send event notifications. You can find your `KEEN_PROJECT_ID` and `KEEN_API_WRITE_KEY` listed in the [Access tab](https://keen.io/projects) of your Keen project.
-
-```
-https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v3?api_key=KEEN_API_WRITE_KEY
-```
-
-If you're using Version 1 or 2 of the SendGrid Event API, use this as your Post Event URL instead:
-
-```
-https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY
-
- ```
-
-In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check "Select All" under the "Select Actions", or select just the ones you'd like to send to Keen IO.
+1. First click on <a href="http://app.sendgrid.com/settings/mail_settings">Mail Settings</a> within Settings in the left column. 
+2. Next, locate and activate the Event Notification Setting.
+   ![SendGrid- Mail Settings]({{root_url}}/images/keen/sendgrid_mailsettings_eventnotification.png)
+3. Enter the Keen HTTP POST URL, exampled below, as the endpoint where SendGrid will send event notifications. You can find your `KEEN_PROJECT_ID` and `KEEN_API_WRITE_KEY` listed in the [Access tab](https://keen.io/projects) of your Keen project.
+    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v3?api_key=KEEN_API_WRITE_KEY```
+    If you're using Version 1 or 2 of the SendGrid Event API, use this as your Post Event URL instead:
+    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY```
+4. In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check "Select All" under the "Select Actions", or select just the ones you'd like to send to Keen IO.
 
 That’s it! Now, as your emails make their way through SendGrid, all of the event information will be posted to Keen IO. A new Event Collection, aptly named "SendGrid Email Events", will be created within your project.
 
-
-**Step 3: Send a test email!**
+{% anchor h4 %}
+Step 3: Send a test email!
+{% endanchor h4 %}
 
 To test this out, you can click on the “Test Your Integration” button in the Event Notification settings inside of SendGrid, which should create a set of sample email events from SendGrid into your Keen IO Project.
```

Comment:
 
--
To test this out, click **Test Your Integration** in the SendGrid Event Notification settings.  This creates a set of sample email events from SendGrid into your Keen IO Project.
 